### DISCUSSION TITLE
加入回收站功能

### DISCUSSION THREAD

**Discussion by 201202228117** [Meloong-Git/PCL#3548] [讨论]
删除的版本先在回收站存15天，然后在正式删除

**Comment by allMagicNB** [comment]
目前删除版本本来就会移动文件夹到系统的回收站……

**Comment by 201202228117** [comment]
谢谢







---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年3月21日 20:21 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [Hex-Dragon/PCL2] 加入回收站功能 (Discussion #3548) |

目前删除版本本来就会移动文件夹到系统的回收站……
